etc/config/c++.amazon.properties

```diff
@@ -276,6 +276,24 @@ compiler.arm710.semver=7.2.1
 compiler.arm64g630.exe=/opt/compiler-explorer/arm64/gcc-6.3.0/aarch64-unknown-linux-gnueabi/bin/aarch64-unknown-linux-gnueabi-g++
 compiler.arm64g630.name=ARM64 gcc 6.3.0 (linux)
 compiler.arm64g630.semver=6.3.0
+compiler.armg640.exe=/opt/compiler-explorer/arm/gcc-6.4.0/arm-unknown-linux-gnueabi/bin/arm-unknown-linux-gnueabi-g++
+compiler.armg640.name=ARM gcc 6.4
+compiler.armg640.semver=6.4.0
+compiler.armg730.exe=/opt/compiler-explorer/arm/gcc-7.3.0/arm-unknown-linux-gnueabi/bin/arm-unknown-linux-gnueabi-g++
+compiler.armg730.name=ARM gcc 7.3
+compiler.armg640.semver=7.3.0
```

Comment:
typo here! will merge and fix

Reply:
Oh boy, I've introduced more copy'paste errors than just this one line. There are 4 of this kind in the `c.amazon.properties` and 2 or 3 more in `c++.amazon.properties`. Sorry.

Reply:
No worries at all! I think I got them all :)

Now live on beta! https://godbolt.org/beta/

Reply:
Thanks for this!

I'm afraid there is yet another typo. You caught the `group.gccaarch64` one, but I smuggled a `;` in the `group.cross` where a `:` should be. This probably messes up the ARM64 and Power compilers.
Again, sorry for the terrible copy'paste.

Also, when I look at the compilers for C or C++, the latest version is always first, for Fortran always last. What have I done to cause this?